### ISSUE TITLE
[Bug]: "Use color scheme font" not persisted

### ISSUE THREAD

**Issue by timkelty** [mallowigi/material-theme-issues#210]
### Describe the bug

The "Use color scheme font instead of the default" setting is not persisted when quitting PHPStorm.

<img width="550" alt="CleanShot 2022-08-15 at 11 08 00@2x" src="https://user-images.githubusercontent.com/18329/184661560-3d963ac8-8445-40b9-8a13-f3656fea027f.png">

It is always checked when launching after quitting, and the fonts are reset.

This happened recently, either with PHPstorm 2022.2, or a related update to the Material UI plugin

### To Reproduce

1. Uncheck "Use color scheme font instead of the default"
2. Click Apply
3. Verify your fonts are using your default setting
4. Quick PHPStorm
5. Go to "Color Scheme Font" in settings
6. the setting will be checked, and fonts will be using the color scheme font

### Expected behavior

Setting should persist

### Material Theme Version

7.8.0

### IDE

PHPStorm

### System Info

```shell
- MacOS 12.5 (21G72)
- PHPstorm 2022.2
```

### Logs/Screenshots

```shell
https://share.cleanshot.com/huHyUV
```

**Comment by timkelty** [comment]
On second thought this might just be confusing UI from PHPStorm's side…

If I make that change, and then duplicate the scheme to my own, it seems to persist.

**Comment by mallowigi** [comment]
https://material-theme.com/docs/configuration/features-settings/#use-editor-default-font

TLDR: try to see if disabling this checkbox fixes the issue

**Comment by timkelty** [comment]
Thanks!
